python/ray/monitor.py

```diff
@@ -478,6 +482,104 @@ def driver_removed_handler(self, unused_channel, data):
 
         self._clean_up_entries_for_driver(driver_id)
 
+    def _xray_clean_up_entries_for_driver(self, driver_id):
```

Comment:
I wonder if these methods should just call `state.task_table()` and `state.object_table()`? What would be the downside of doing that?

Reply:
It looks like it would be more costly, but it might make sense as a separate PR for long-term consistency.

Reply:
Done.